spec/integration/post_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system, js: true do
+  describe 'index page' do
+    before(:example) do
+      @messi = User.create(name: 'Messi', photo: 'https://i.imgur.com/messi.jpg',
+                           bio: 'I love Barcelona and Argentina.')
+      @cr7 = User.create(name: 'Cristiano', photo: 'https://i.imgur.com/cr7.jpg',
+                         bio: 'I am ubleivable inside the pitch.')
+
+      @messi_post1 = Post.create(title: 'First Post', text: 'This is the first post.', author: @messi)
+      @messi_post2 = Post.create(title: 'Second Post', text: 'This is the second post.', author: @messi)
+      @messi_post3 = Post.create(title: 'Third Post', text: 'This is the third post.', author: @messi)
+      @messi_post4 = Post.create(title: 'Fourth Post', text: 'This is the fourth post.', author: @messi)
+
+      @comment1 = Comment.create(text: 'Wassup G!.', author: @cr7, post: @messi_post2)
+      @comment2 = Comment.create(text: "I'm not feeling too good, G!.", author: @messi, post: @messi_post2)
+      @comment3 = Comment.create(text: 'How come, G?.', author: @cr7, post: @messi_post2)
+      @comment4 = Comment.create(text: 'I miss Iniesta and xavi, G!.', author: @messi, post: @messi_post2)
+      @comment5 = Comment.create(text: 'I miss Ramos and Marcelo, G!.', author: @cr7, post: @messi_post2)
+      @comment6 = Comment.create(text: 'I miss the old days, G!.', author: @messi, post: @messi_post1)
+      @comment7 = Comment.create(text: 'I miss the old days too, G!.', author: @cr7, post: @messi_post3)
+      @comment8 = Comment.create(text: 'I miss the old days too, G!.', author: @messi, post: @messi_post4)
+
+      visit user_posts_path(@messi)
+    end
+
+    it 'should render user profile information' do
+      expect(page).to have_css("img[src*='messi.jpg']")
+      expect(page).to have_content(@messi.name)
+      expect(page).to have_content("Posts(#{@messi.posts_counter})")
+    end
+
+    it 'should render user posts' do
```

Comment:
- [ ] Please fix this 26 errors and make all the tests pass as a professional developer. I suggest you to remove cleaner and webdrivers and only use rspec and capybara if you couldn't make it work as it's not a requirement.
![image](https://github.com/serengia/rails-blog-app/assets/91486791/72e1ef2f-0436-4533-baf2-0b1132d05316)